doc/deployment/deploy-to-azure-web-app-tomcat-using-jenkins.md

```diff
@@ -0,0 +1,57 @@
+# Deploy to Azure Web App for Tomcat using Jenkins
```

Comment:
I am not sure with this title. Since  this todo-app using Spring boot built-in tomcat. And this article tells how to build a war and deploy this war to app service. Nothing explicitly with Tomcat. Maybe here need @Matthew-Dong 's input.

Reply:
Agree. Somethings to decide before writing this doc:

1) Use WAR or JAR as the package type? Usually a Spring Boot app is packaged as a Jar application with built-in Tomcat. 
2) If using JAR, the App Service Linux supports Java SE running environment.
3) If using JAR, the Jenkins plugin supports deploying with ZIP Deploy.